.github/workflows/asset_dispatch.yml

```diff
@@ -23,13 +30,25 @@ jobs:
     - name: Compile Build
       run: ./compile-build
 
+    - name: Create Release
```

Comment:
This action is not actively maintained anymore, let's see how [this](https://github.com/sanger/traction-ui/pull/592) goes and then implement that logic here?

Reply:
Yip, seems like we can combine the release creation and upload with that done in traction-ui.